index.html

```diff
@@ -2573,18 +2603,23 @@ <h5>Uses beyond mapping</h5>
 
 <h5>Related web specifications</h5>
 <p>
-<!-- Any existing/proposed specs that address similar needs or otherwise interact.
-      Include links with proper ReSpec references. -->
+The <a data-cite="SVG2#">SVG</a> <code>image</code> element provides an example
+of external image content being rendered into a specified rectangle within a coordinate system.
+</p>
+<p>
+[[[Canvas]]] includes a JavaScript method, <code>drawImage()</code>
+for embedding external images within the canvas element.
 </p>
-
 <h5>Conclusion</h5>
 <p>
 </p>
 <ul data-ucr-role="conclusion-notes">
 <!-- bullet-point notes summarizing implementation details / blocking issues.
       Add tags to the start of a point with an empty `<a data-ucr-role="tag"></a>` where the text content or href matches a tag `id` value. -->
-<li><a data-ucr-role="tag" href=""></a>
-  <!-- short summary of how the tag applies -->
+<li>
+<a data-ucr-role="tag" href="#implementation-incremental"></a>
+As described above, both SVG and Canvas are examples of existing web specifications
```

Comment:
This might be pushing it. It's incremental in that browsers can already render images, but that's also in the introduction: " in a standard image format the browser can render". I'm not sure there's any reasonable expectation that map coordinate / positioning code is going to build upon SVG/Canvas coordinates.

Reply:
Understood. Should I also remove the reference to Canvas/SVG in the related specifications part? Or just this tag?

Reply:
They're still related specifications, that should be considered from a design perspective. Just unlikely to make implementation much easier.

Reply:
Removed this tag.